reporters/GlobalSummaryReporter.js

```diff
@@ -295,10 +291,33 @@ class GlobalSummaryReporter {
       current: this.formatDecimalString(this.totalTokensOutstanding)
     };
 
-    // TODO:
-    // - # token holders (current) (cumulative)
-    // - # trades in uniswap (24H) (cumulative)
-    // - volume of trades in uniswap in # of tokens (24H) (cumulative)
+    // Get uniswap data via graphql.
+    // TODO: Programmatically get this pair address.
```

Comment:
@mrice32 guessing you would know better how I can programmatically grab this?

Reply:
I would use this function: https://github.com/UMAprotocol/protocol/blob/004db01a50eaad2176069676463610037b88f71a/financial-templates-lib/price-feed/CreatePriceFeed.js#L127-L147. Feel free to move it to `common/` since we're using it in multiple places.

Reply:
Done. Eventually we should move `getUniswapPairDetails` along with the uniswap subgraph modules into their own `utils`-like directory for other apps to access